<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VueConnexion</class>
 <widget class="QDialog" name="VueConnexion">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/Images/Toolbox.ico</normaloff>:/Images/Toolbox.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="etiquettePort">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="etiquetteMDP">
       <property name="text">
        <string>Mot de passe:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="etiquetteHote">
       <property name="text">
        <string>HÃ´te:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="etiquetteUsager">
       <property name="text">
        <string>Usager:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="champHote">
       <property name="placeholderText">
        <string>ex. localhost, 192.168.0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="champPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>3306</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="champUsager"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="champMDP">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Aucun</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="boutonConnecter">
     <property name="text">
      <string>Connecter</string>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/Images/network-connect.svg</normaloff>:/Images/network-connect.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="listeBd">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>champHote</tabstop>
  <tabstop>champPort</tabstop>
  <tabstop>champUsager</tabstop>
  <tabstop>champMDP</tabstop>
  <tabstop>boutonConnecter</tabstop>
  <tabstop>listeBd</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VueConnexion</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VueConnexion</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
